cmd/osm-bootstrap/osm-bootstrap_test.go

```diff
@@ -43,7 +43,7 @@ var testPresetMeshConfigMap *corev1.ConfigMap = &corev1.ConfigMap{
 	"logLevel": "error",
 	"maxDataPlaneConnections": 0,
 	"envoyImage": "envoyproxy/envoy-alpine:v1.19.3@sha256:874e699857e023d9234b10ffc5af39ccfc9011feab89638e56ac4042ecd4b0f3",
-	"initContainerImage": "openservicemesh/init:latest-main",
+	"initContainerImage": "openservicemesh/init:v1.1.0",
```

Comment:
This test will still pass as-is, so we don't need to change this. If the test passes without this line at all, I'd also be in favor of removing it.